app/controllers/npq_separation/admin/finance/statements/adjustments_controller.rb

```diff
@@ -2,19 +2,27 @@
 
 class NpqSeparation::Admin::Finance::Statements::AdjustmentsController < NpqSeparation::AdminController
   before_action :set_statement
+  before_action :find_adjustment, only: %i[edit update delete destroy]
+  before_action :set_show_all_adjustments, except: %i[create add_another]
 
   def new
     @adjustment = @statement.adjustments.new
+    @cancel_url = if @show_all_adjustments
+                    npq_separation_admin_finance_statement_adjustments_path(@statement, show_all_adjustments: @show_all_adjustments)
+                  else
+                    npq_separation_admin_finance_statement_path(@statement)
+                  end
```

Comment:
Could just add a private method in the controller and mark it using `helper_method :cancel_url` ?

Reply:
maybe - the values of @cancel_url are different for the `new` and `edit` methods - so the helper method would end up being `cancel_new_url`?
I'm not sure it's worth creating a helper method when it's only used once.